packages/office-ui-fabric-react/src/components/CommandBar/examples/CommandBar.ButtonAs.Example.tsx

```diff
@@ -9,6 +9,9 @@ import {
   IContextualMenuStyles,
 } from 'office-ui-fabric-react/lib/ContextualMenu';
 import { getTheme } from 'office-ui-fabric-react/lib/Styling';
+import { IButtonStyles } from '../../Button';
+import { concatStyleSets } from '@uifabric/styling';
+import { memoizeFunction } from 'office-ui-fabric-react/lib/Utilities';
```

Comment:
Imports in examples need to be absolute and should be from OUFR if possible. (These should have all caused errors with the lint-imports script, so if they're not, I'd like to investigate that.)
```suggestion
import { getTheme, concatStyleSets } from 'office-ui-fabric-react/lib/Styling';
import { IButtonStyles } from 'office-ui-fabric-react/lib/Button';
import { memoizeFunction } from 'office-ui-fabric-react/lib/Utilities';
```